src/responder/nss/nsssrv_mmap_cache.c

```diff
@@ -1352,8 +1352,8 @@ errno_t sss_mmap_cache_init(TALLOC_CTX *mem_ctx, const char *name,
         return EOK;
     }
     DEBUG(SSSDBG_CONF_SETTINGS,
-          "Fast '%s' mmap cache: memcache_timeout = %d, slots = %zu\n",
-          mc_type_to_str(type), (int)timeout, n_elem);
+          "Fast '%s' mmap cache: memcache_timeout = %ld, slots = %zu\n",
```

Comment:
Hi,

I wonder if this might cause warnings when compiling on a 32bit platform where time_t is still 32bit as well. Maybe it would be worth add a SPRItimet format to src/util/sss_format.h to have a proper format for printing time_t values?

bye,
Sumit

Reply:
Good idea!